src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -571,6 +570,7 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
     <RemoveDirFixed Directories="$(IntermediateOutputPath)library_project_annotations" Condition="Exists ('$(IntermediateOutputPath)library_project_annotations')" />
     <RemoveDirFixed Directories="$(IntermediateOutputPath)$(_LibraryProjectImportsDirectoryName)" Condition="Exists ('$(IntermediateOutputPath)$(_LibraryProjectImportsDirectoryName)')" />
     <RemoveDirFixed Directories="$(IntermediateOutputPath)__library_projects__" Condition="Exists ('$(IntermediateOutputPath)__library_projects__')"/>
+	<RemoveDirFixed Directories="$(_AndroidLibrayProjectIntermediatePath)" Condition="Exists ('$(_AndroidLibrayProjectIntermediatePath)')" />
```

Comment:
Indentation here doesn't match surrounding text.

(The perils of a single file that uses both tabs and spaces...)